doc/governance/hsm_keys.rst

```diff
@@ -117,45 +117,6 @@ Like ``ccf_cose_sign1``, the output can be sent directly to the service via curl
         "state": "Open"
     }
 
-HTTP Signing
```

Comment:
I think we may want to keep this section with a warning that this scheme was removed in 4.x.